openshift/gabi-no-cluster-resources.template.yaml

```diff
@@ -170,7 +170,7 @@ objects:
 - apiVersion: v1
   kind: Service
   metadata:
-    name: ${GABI_INSTANCE}
+    name: ${GABI_INSTANCE}-tls
```

Comment:
`${GABI_INSTANCE}-tls` is not for Service name, it is for the annotations service.alpha.openshift.io/serving-cert-secret-name's value